consensus/src/round_manager.rs

```diff
@@ -318,10 +317,9 @@ impl RoundManager {
             sync_info
                 .verify(&self.epoch_state().verifier)
                 .map_err(|e| {
-                    security_log(SecurityEvent::InvalidSyncInfoMsg)
-                        .error(&e)
-                        .data(&sync_info)
-                        .log();
+                    send_struct_log!(security_log(security_events::INVALID_SYNC_INFO_MSG)
+                        .data("sync_info", &sync_info)
```

Comment:
Btw, if you _want_ you can consider using fields.

Example declaration:
https://github.com/libra/libra/blob/51e6e69c8a8b68e0087fb43c93ab89ad85aa0769/network/src/logging.rs#L25
Usage:
https://github.com/libra/libra/blob/ebecc430e7f78d0968fdcf188555cf3b27b79cfc/network/onchain-discovery/src/client.rs#L131

Reply:
I was wondering about that, this is specifically when the same log is used in different places right?